lms/djangoapps/mobile_api/users/tests.py

```diff
@@ -205,6 +205,15 @@ def test_facebook_url(self):
         course_data = response.data[0]['course']
         self.assertEquals(course_data['social_urls']['facebook'], self.course.facebook_url)
 
+    @patch.dict("django.conf.settings.FEATURES", {"ENABLE_DISCUSSION_SERVICE": True})
+    def test_discussion_url(self):
+        expected_discussion_url = 'http://testserver/api/mobile/v0.5/discussion_course/v1/courses/%s' % self.course.id
```

Comment:
There's no need to hard-code the base server domain name, just in case we need to change this in our test infrastructure.

You can change the assertion below to ensure the minimum of what we're testing.  For example:
`self.assertIn('/api/discussion', course_data['discussion_url'])`